packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -396,6 +397,20 @@ class ApplicationCommand extends Base {
     }
     return true;
   }
+  
+  /**
+   * When concatenated with a string, if this is a chat input command,
+   * this automatically returns the command's mention instead of the ApplicationCommand object.
```

Comment:
The requirement would be better seen in `<info>`:

```suggestion
   * When concatenated with a string, this automatically returns the command's mention
   * instead of the ApplicationCommand object.
   * <info>This only works with {@link ChatInputCommandInteraction}s.</info>
```

Reply:
This is not an interaction. I don't think `ChatInputCommandInteraction` is appropriate here.